lib/fxa.js

```diff
@@ -53,7 +72,20 @@ const FXA = {
     await this.destroyOAuthToken({ refresh_token: subscriber.fxa_refresh_token });
   },
 
+  async getProfileData(accessToken) {
+    const data = await got(FxAOAuthUtils.profileUri,
+      {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    });
+    return data.body;
```

Comment:
Should this return a string, or should we be returning a JSON blob?

Reply:
I'm returning the json just like the `confirmed` controller code did before. Trying to minimize the code changes in this PR, and also because the code puts this data in the `fxa_profile_json` column in the DB, which is `jsonb` type.

Reply:
Hhmm, then I'm a bit confused by 

https://github.com/mozilla/blurts-server/pull/1352/files#diff-075ff2d4bc68f5a42b68daf27075ab5bR49-R51

```js
const fxaProfileData = await FXA.getProfileData(fxaUser.accessToken);
log.debug("fxa-confirmed-profile-data", fxaProfileData);
const email = JSON.parse(fxaProfileData).email;
```

If we're returning a JSON object, I thought that we wouldn't need the extra `JSON.parse()` bit there. But let me try pulling the PR locally and add some console loggin' and see why I'm confused. I thought if you needed to specify [`{json: true}`](https://www.npmjs.com/package/got#json) for **`got.get()`** if you wanted Got to auto `JSON.parse()` the response.